docs/source/api/apollo-server.md

```diff
@@ -345,10 +345,26 @@ addMockFunctionsToSchema({
 
 *  `privateVariables`: Array<String> | boolean
 
+   DEPRECATING IN VERSION XX.XX.XX for `enforcePrivateVariables`, which will support the same
+   functionalities but allow for more flexibility.
+   
    A case-sensitive list of names of variables whose values should not be sent
    to Apollo servers, or 'true' to leave out all variables. In the former
    case, the report will indicate that each private variable was redacted in
    the latter case, no variables are sent at all.
+   
+* `enforcePrivateVariables`: ((variables: Record<string, any>, operationString?: string) => Record<string, any>) | Array<String> | boolean
+    
+    A custom function for modifying variable values, a case-sensitive list of names 
```

Comment:
The first two sentences here are pretty long and require the reader to carefully map between their comma-separated clauses. I'd start with an overall description of what this is about including a motivation, and then enumerate the cases, maybe in a bulleted list (or a `<dl>` if that looks good in our docs theme). Something like:

> By default, Apollo Server doesn't send the values of any GraphQL variables to Apollo's servers, because variable values often contain the private data of your app's users. If you'd like variable values to be included in traces, set this option. This option can take several forms:
> - `true`: ...
> - a list of variables names
> - etc

Reply:
👍